atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -65,7 +65,7 @@ default CassandraServersConfig servers() {
         return ImmutableDefaultConfig.of();
     }
 
-    // Todo(snanda): the field is no longer in use
+    @JsonIgnore
```

Comment:
Noting that this _is_ used in the internal product - can these usages also be removed?

If so - then we can remove the field in AtlasDB (after removing the usages internally)
If not - then we shouldn't ignore it.